src/main/java/core/basesyntax/Properties.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Properties {
+    double countArea();
+
+    void getInfo();
+}
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-put-all-behaviour-into-a-single-interface-if-the-methods-are-conceptually-different-from-each-other

create 2 separate interfaces AreaCalculator and FigurePrinter